src/components/CommentItem.jsx

```diff
@@ -14,8 +14,8 @@ const propTypes = {
     timestamp: PropTypes.number,
     created_by: PropTypes.number,
     username: PropTypes.string,
-    n_likes: PropTypes.number,
-    liked: PropTypes.number,
+    n_likes: PropTypes.string,
```

Comment:
This wrong data type from the DB was the reason for my last PR at the server. To cast the type to integer directly in the query. But this also works

Reply:
Yeah, it works without casting it, so I don't think it's necessary.

Reply:
should really be treated as integers, since:
eg.
this.setState({ nLikes: nLikes + 1, liked: 1 });

Can also be casted at client side ofc where:
    this.state = {
      nLikes: comment.n_likes,
      liked: comment.liked,
    };

Reply:
Good point. I'll merge your change on the API.